src/PatternBuilder.ts

```diff
@@ -0,0 +1,269 @@
+import * as fc from 'fast-check'
```

Comment:
🔧  I think we might add some more JSDoc comments in this file, especially if we intend to make it user facing.  And also, docs-ts might fail if there's no JSDoc at the top of the file.

Reply:
We can also save that for later too, I'm okay with it either way